rotkehlchen/tests/api/test_user_assets.py

```diff
@@ -339,7 +362,7 @@ def test_editing_user_assets(rotkehlchen_api_server, globaldb):
     )
     # try to add invalid cryptocompare
     bad_id = 'dsadsad'
-    bad_asset = {
+    evm_token_data = {
```

Comment:
why rename it to `evm_token_data`? is not an evm token. Same to similar changes in this test

Reply:
Thanks for noticing it. It got autorenamed unexpectedly. I meant to rename only one such dict, where there is evm token data, but since they all weer named `bad_asset`, all of them were renamed